lib/controllers/src/ActuatorController.cpp

```diff
@@ -217,4 +219,133 @@ bool ActuatorController::isPastMinAngle() {
 
 bool ActuatorController::isPastMaxAngle() {
   return (getAngle_Degrees() > m_actuatorConfig.maxAngle_Degrees || isLimSwitchMaxTriggered());
-}
\ No newline at end of file
+}
+
+// Update m_actuatorConfig upon reception of new PID values
+void ActuatorController::updateActuatorConfigP(const float newP, const bool isVelocityPID) {
+  isVelocityPID ? m_actuatorConfig.velocityPID.P : m_actuatorConfig.positionPID.P = newP;
```

Comment:
I think the isVelocityPID bool should be updated to an enum or something similar, considering we want to add a torque/current PID controller to the ActuatorController class in the future. It would also improve readability.